src/RepoM.ActionMenu.Core/RepoM.ActionMenu.Core.csproj

```diff
@@ -9,7 +9,7 @@
     <PackageReference Include="Scriban" Version="5.10.0" />
     <PackageReference Include="SimpleInjector" Version="5.5.0" />
     <PackageReference Include="System.IO.Abstractions" Version="21.0.29" />
-    <PackageReference Include="YamlDotNet" Version="16.0.0" />
+    <PackageReference Include="YamlDotNet" Version="16.1.0" />
```

Comment:
Was it necessary to update YamlDotNet? If not, please don't. For some packages I like to read the changelog before updating.

Reply:
Not necessary, but noticed some performance improvements introduced in version 16.1.0 : https://github.com/aaubry/YamlDotNet/releases

I'll leave it up to you whether you want to update to it or not.